packages/commonwealth/server/scripts/setupChainEventListeners.ts

```diff
@@ -189,7 +191,9 @@ const setupChainEventListeners = async (
             contractVersion,
           });
         } else if (node.network === ChainNetwork.Compound) {
-          const api = await CompoundEvents.createApi(node.ChainNode.url, node.address);
+          // @TODO: @JAKE @CONTRACTS: fetch governance contract specifically
+          const contracts = await node.getContracts(); // should 'includes' ChainNode from contract too
```

Comment:
May need to specify a "requires" in the call if you want to guarantee it grabs ChainNode.

Reply:
we don't use it here, because we defer to the chain's node as `node.ChainNode.url` instead (always fine currently). gonna remove the comment